server/workflow.go

```diff
@@ -24,19 +24,19 @@ func NewTriggerStore() *TriggerStore {
 	}
 }
 
-func (t TriggerStore) AddTrigger(trigger WorkflowTrigger, callbackURL string) {
+func (t *TriggerStore) AddTrigger(trigger WorkflowTrigger, callbackURL string) {
```

Comment:
FWIW, I've been trying for a long time to prefer value receivers, because it _seems_ the right thing to do, and I am giving up. Too confusing/error prone. They really ought to be reserved for special/edge cases. Please feel free to remind me of it if you ever see me do otherwise!